src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/index.tsx

```diff
@@ -0,0 +1,197 @@
+import { AzureResourceGroups } from './AzureResourceGroups';
+import { AzureSourcesSelect } from './AzureSourcesSelect';
+
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  changeAzureResourceGroup,
+  changeAzureShareMethod,
+  changeAzureSource,
+  changeAzureSubscriptionId,
+  changeAzureTenantId,
+  selectAzureResourceGroup,
+  selectAzureShareMethod,
+  selectAzureSubscriptionId,
+  selectAzureTenantId,
+} from '../../../../../store/wizardSlice';
+import { ValidatedTextInput } from '../../../ValidatedTextInput';
+import {
+  isAzureResourceGroupValid,
+  isAzureSubscriptionIdValid,
+  isAzureTenantGUIDValid,
+} from '../../../validators';
+
+export type AzureShareMethod = 'manual' | 'sources';
+
+const SourcesButton = () => {
+  return (
+    <Button
+      component="a"
+      target="_blank"
+      variant="link"
+      icon={<ExternalLinkAltIcon />}
+      iconPosition="right"
+      isInline
+      href={'settings/sources'}
+    >
+      Create and manage sources here
+    </Button>
+  );
+};
+
+const Azure = () => {
+  const dispatch = useAppDispatch();
+  const shareMethod = useAppSelector((state) => selectAzureShareMethod(state));
+  const tenantId = useAppSelector((state) => selectAzureTenantId(state));
+  const subscriptionId = useAppSelector((state) =>
+    selectAzureSubscriptionId(state)
+  );
+  const resourceGroup = useAppSelector((state) =>
+    selectAzureResourceGroup(state)
+  );
+
+  return (
+    <Form>
+      <Title headingLevel="h2">Target environment - Microsoft Azure</Title>
+      <Text>
+        Upon build, Image Builder sends the image to the selected authorized
+        Azure account. The image will be uploaded to the resource group in the
+        subscription you specify.
+      </Text>
+      <Text>
+        To authorize Image Builder to push images to Microsoft Azure, the
+        account owner must configure Image Builder as an authorized application
+        for a specific tenant ID and give it the role of &quot;Contributor&quot;
+        for the resource group you want to upload to. This applies even when
+        defining target by Source selection.
+        <br />
+        <Button
+          component="a"
+          target="_blank"
+          variant="link"
+          icon={<ExternalLinkAltIcon />}
+          iconPosition="right"
+          isInline
+          href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow"
+        >
+          Learn more about OAuth 2.0
+        </Button>
+      </Text>
+      <FormGroup label="Share method:">
+        <Radio
+          id="radio-with-description"
+          label="Use an account configured from Sources."
+          name="radio-7"
+          description="Use a configured sources to launch environments directly from the console."
+          isChecked={shareMethod === 'sources'}
+          onChange={() => {
+            dispatch(changeAzureSource(undefined));
```

Comment:
We'll need to make this `''` for the empty state if we change it in the wizardSlice.